terraform/paas/production.env.tfvars

```diff
@@ -1,11 +1,11 @@
-paas_space                 = "get-into-teaching-production"
-paas_app_route_name        = "get-into-teaching-app-prod"
-paas_app_application_name  = "get-into-teaching-app-prod"
-paas_redis_1_name          = "get-into-teaching-prod-redis-svc"
-paas_additional_route_name = "beta-getintoteaching"
-instances                  = 2
-azure_key_vault            = "s146p01-kv"
-azure_resource_group       = "s146p01-rg"
+paas_space                  = "get-into-teaching-production"
+paas_app_route_name         = "get-into-teaching-app-prod"
+paas_app_application_name   = "get-into-teaching-app-prod"
+paas_redis_1_name           = "get-into-teaching-prod-redis-svc"
+paas_additional_route_names = ["beta-getintoteaching", "getintoteaching"]
```

Comment:
`paas_additional_route_names` is a bit misleading. Should it be something like `internet_hostnames`?

Reply:
changed to paas_internet_hostnames